docs/modules/ROOT/pages/installation.adoc

```diff
@@ -57,10 +57,10 @@ docker run \
 
 == Building the operator from source
 
-This operator is written in Rust and is developed against the latest stable Rust release (1.57 at the time of writing).
+This operator is written in Rust and is developed against the latest stable Rust release (1.60 at the time of writing).
 
-[source]
+[source,bash]
 ----
-stackable-hive-operator crd | kubectl apply -f -
-stackable-hive-operator run
+cargo run stackable-hive-operator crd | kubectl apply -f -
```

Comment:
On my machine, this doesn't actually run. I think what you mean is:

    cargo run crd | kubectl apply -f -
    cargo run run

Reply:
Thank you! Fixed now.